added some functions in the content.py file

Biruk-Tassew opened the pull request.

GizawAAiT commented:
-  Please use descriptive name for your function like `def product(x, y): `  instead of  `def abc(x, y): `

Biruk-Tassew commented:
> * Please use descriptive name for your function like `def product(x, y): `  instead of  `def abc(x, y): `

I've updated what you required and added some comments.
